File: app/src/main/java/com/ntech/fourtop/network/pojo/DataProfile.java
package com.ntech.fourtop.network.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DataProfile {

    @SerializedName("tester_name")
    @Expose
    private String testerName;
    @SerializedName("login")
    @Expose
    private String login;
    @SerializedName("email")
    @Expose
    private String email;

    public String getTesterName() {
        return testerName;
    }

    public void setTesterName(String testerName) {
        this.testerName = testerName;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: app/src/main/java/com/emikhalets/voteapp/network/pojo/DataProfile.kt
package com.emikhalets.voteapp.network.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class DataProfile {
    @SerializedName("tester_name")
    @Expose
    var testerName: String? = null

    @SerializedName("login")
    @Expose
    var login: String? = null

    @SerializedName("email")
    @Expose
    var email: String? = null
}